registry/remote/url.go

```diff
@@ -99,3 +98,16 @@ func buildArtifactReferrerURL(plainHTTP bool, ref registry.Reference) string {
 		ref.Reference,
 	)
 }
+
+// buildArtifactReferrerURL builds the URL for accessing the manifest referrers API in artifact spec v1.0.0-rc.1.
+// Format: <scheme>://<registry>/v2/<repository>/_oras/artifacts/referrers?digest=<digest>
+// Reference: https://github.com/oras-project/artifacts-spec/blob/v1.0.0-rc.1/manifest-referrers-api.md
+func buildArtifactReferrerURL(plainHTTP bool, ref registry.Reference) string {
+	return fmt.Sprintf(
+		"%s://%s/v2/%s/_oras/artifacts/referrers?digest=%s",
+		buildScheme(plainHTTP),
+		ref.Host(),
+		ref.Repository,
```

Comment:
The referrer API in the latest spec falls in the `/v2/` path. Should we leverage `buildRepositoryBaseURL` in `buildArtifactReferrerURL`?